src/navigation/underline-nav.scss

```diff
@@ -21,30 +30,62 @@
   white-space: nowrap;
   background-color: transparent;
   border: 0;
-  // stylelint-disable-next-line primer/borders
-  border-bottom: 2px $border-style transparent;
+  border-radius: $border-radius;
+  align-items: center;
 
-  &:hover,
-  &:focus {
-    color: var(--color-fg-default);
-    text-decoration: none;
-    border-bottom-color: var(--color-neutral-muted);
-    outline: 1px dotted transparent; // Support Firefox custom colors
-    outline-offset: -1px;
-    transition: border-bottom-color 0.12s ease-out;
+  // increase touch target area
+  &::before {
+    @include minTouchTarget($min-height: $nav-height);
+  }
+
+  // hover state was "sticking" on mobile after click
+  @media (pointer: fine) {
+    &:hover {
+      color: var(--color-fg-default);
+      text-decoration: none;
+      background: var(--color-neutral-muted);
```

Comment:
Is this intentional `border-bottom-color` -> `background`? It gets overridden below. 👇  I guess this line can be removed if no border is needed.

```suggestion
```

Reply:
I'm thinking about leaving the action-list primitive in, and opening an issue to create new primitives for this kind of interaction (not action-list specific)

I didn't feel like the hover underline was obvious enough (I actually didn't even notice it) so the background replaces it. Open to suggestions!

Reply:
> I didn't feel like the hover underline was obvious enough (I actually didn't even notice it) so the background replaces it. Open to suggestions!

Ahh.. yes, I think using background for hover is great. 👍  I just meant that the `background` is repeated two times and the first is probably not needed.